### PR TITLE
Update teams-toolkit-fundamentals.md

### PR DESCRIPTION
**Pull request by SPDoctor**
A few typos, and polished up the English so it reads a bit better.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/teams-toolkit-fundamentals.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d15cef98478e34b9deb7eeb95879bfe60bd1d1a1/msteams-platform/toolkit/teams-toolkit-fundamentals.md) | 85 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fd1e1411-68e3-4b1c-a708-31e1460aba9f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/teams-toolkit-fundamentals.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ ms.date: 11/29/2021
 > [!NOTE]
 > Currently, this feature is available in **public developer preview** only.
 
-Teams Toolkit for Visual Studio Code helps you to create and deploy Teams apps with integrated identity, access to cloud storage, data from Microsoft Graph, and other services in Azure and Microsoft 365 with zero-configuration approach. For Teams app development, similar to Teams Toolkit for Visual Studio, you can use [CLI tool](https://github.com/OfficeDev/TeamsFx/blob/dev/docs/cli/user-manual.md), which consists of Toolkit `teamsfx`.
+The Teams Toolkit for Visual Studio Code helps you to create and deploy Teams apps with integrated identity, access to cloud storage, data from Microsoft Graph, and other services in Azure and Microsoft 365 with a zero-configuration approach. For Teams app development from the command line you can also use the Teams Toolkit through the `TeamsFx` [CLI tool](https://github.com/OfficeDev/TeamsFx/blob/dev/docs/cli/user-manual.md).
```
Comment (v-abirade) [inline msteams-platform/toolkit/teams-toolkit-fundamentals.md:16]: For Teams app development from the command line you can also use the Teams Toolkit through `TeamsFx` [CLI tool](https://github.com/OfficeDev/TeamsFx/blob/dev/docs/cli/user-manual.md).

File: msteams-platform/toolkit/teams-toolkit-fundamentals.md [hunk 2]
```diff
@@ -53,72 +53,72 @@ Let's take a tour of the topics covered in this document:
...
 
 ### TeamsFx Collaboration
 
-It allows developers and project owner to invite other collaborators to the TeamsFx project to debug, provision, and deploy the same TeamsFx project.
+It allows developers and project owners to invite other collaborators to the TeamsFx project to debug, provision, and deploy the project.
 
 ## Development
 
-Teams Toolkit helps you to create and customize your Teams app project that makes the Teams app development work simpler.
+Teams Toolkit helps you to create and customize your Teams app project, and makes simplifies Teams app development.
```
Comment (v-abirade) [inline msteams-platform/toolkit/teams-toolkit-fundamentals.md:68]: Teams Toolkit helps you to create and customize your Teams app project, and simplifies Teams app development.

File: msteams-platform/toolkit/teams-toolkit-fundamentals.md [hunk 3]
```diff
@@ -53,72 +53,72 @@ Let's take a tour of the topics covered in this document:
...
 
 ### Add cloud resources
 
-It helps you to optionally add cloud resources that fits your development needs.
+It helps you to optionally add cloud resources according to your development needs.
 
 ### Edit manifest file 
 
-It helps you to edit the Teams app integration with Teams client.
+It helps you to configure how the Teams app will be integrated with the Teams client.
```
Comment (v-abirade) [inline msteams-platform/toolkit/teams-toolkit-fundamentals.md:84]: It helps you to configure the Teams apps that will be integrated with the Teams client.

File: msteams-platform/toolkit/teams-toolkit-fundamentals.md [hunk 4]
```diff
@@ -53,72 +53,72 @@ Let's take a tour of the topics covered in this document:
...
 
 ## Deployment
 
-During or after the development, ensure to provision, deploy, and publish Teams app before it is accessible to users.
+During or after development make sure to provision, deploy, and publish your Teams app to make it accessible to users.
```
Comment (v-abirade) [inline msteams-platform/toolkit/teams-toolkit-fundamentals.md:88]: During or after development ensure to provision, deploy, and publish your Teams app to make it accessible to users.

File: msteams-platform/toolkit/teams-toolkit-fundamentals.md [hunk 5]
```diff
@@ -53,72 +53,72 @@ Let's take a tour of the topics covered in this document:
...
 
 ### Provision in the cloud
 
-It integrates with Azure resource manager that enables you to provision Azure resources, which your application needs for code approach.
+It integrates with Azure resource manager, enabling you to provision Azure resources needed by your application.
```
Comment (v-abirade) [inline msteams-platform/toolkit/teams-toolkit-fundamentals.md:92]: Please retain the original sentence.

**Review by v-abirade — CHANGES_REQUESTED** [review]
@SPDoctor, request you to update the changes.

File: msteams-platform/toolkit/teams-toolkit-fundamentals.md (hunk 2 above)
Comment (SPDoctor) [reply msteams-platform/toolkit/teams-toolkit-fundamentals.md:68]: good catch :-)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/teams-toolkit-fundamentals.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8e8ad4bff7a2296fbed666496b70d8dfee012450/msteams-platform/toolkit/teams-toolkit-fundamentals.md) | 83 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1135aaf2-5a5f-4ac9-aa11-b4b3e1b5d9a7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/teams-toolkit-fundamentals.md (hunk 5 above)
Comment (SPDoctor) [reply msteams-platform/toolkit/teams-toolkit-fundamentals.md]: Okay, all done.

File: msteams-platform/toolkit/teams-toolkit-fundamentals.md (hunk 2 above)
Comment (v-abirade) [reply msteams-platform/toolkit/teams-toolkit-fundamentals.md]: Thanks

**Review by v-abirade — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**